component/main.tsx

```diff
@@ -16,3 +16,9 @@ const config = {
 firebase.initializeApp(config);
 
 ReactDOM.render(<ClientCore />, document.getElementById("root"));
+
+const storageService = (chrome.extension.getBackgroundPage() as any).BackgroundStorageService;
```

Comment:
I would prefer that any required information that we need from the Chrome's local storage be done via Redux action. This allows us to organize our logic into a frontend component only caring about displaying data and a backend component only caring about retrieval and processing of data.